tla-types/src/main/scala/at/forsyte/apalache/tla/typecheck/passes/EtcTypeCheckerPassImpl.scala

```diff
@@ -3,32 +3,42 @@ package at.forsyte.apalache.tla.typecheck.passes
 import at.forsyte.apalache.infra.passes.{Pass, PassOptions, TlaModuleMixin}
 import at.forsyte.apalache.io.annotations.store.AnnotationStore
 import at.forsyte.apalache.tla.imp.src.SourceStore
+import at.forsyte.apalache.tla.lir.storage.{ChangeListener, SourceLocator}
 import at.forsyte.apalache.tla.lir.transformations.TransformationTracker
-import at.forsyte.apalache.tla.lir.{TypeTag, UID, Untyped}
+import at.forsyte.apalache.tla.lir.{TlaModule, TypeTag, UID, Untyped}
 import at.forsyte.apalache.tla.typecheck.TypeCheckerTool
 import com.google.inject.Inject
 import com.google.inject.name.Named
 import com.typesafe.scalalogging.LazyLogging
 
 class EtcTypeCheckerPassImpl @Inject() (val options: PassOptions, val sourceStore: SourceStore,
-    tracker: TransformationTracker, val annotationStore: AnnotationStore,
+    changeListener: ChangeListener, tracker: TransformationTracker, val annotationStore: AnnotationStore,
     @Named("AfterTypeChecker") val nextPass: Pass with TlaModuleMixin)
     extends EtcTypeCheckerPass with LazyLogging {
 
+  private var outputTlaModule: Option[TlaModule] = None
+
   /**
    * The name of the pass
    *
    * @return the name associated with the pass
    */
-  override def name: String = "TypeChecker"
+  override def name: String = "TypeCheckerSnowcat"
 
   /**
    * Run the pass.
    *
    * @return true, if the pass was successful
    */
   override def execute(): Boolean = {
-    if (tlaModule.isDefined) {
+    if (tlaModule.isEmpty) {
+      logger.info(" > no input for type checker")
+      false
+    } else if (!options.getOrElse("typechecker", "snowcatOn", false)) {
+      logger.info(" > Snowcat is disabled. Use --with-snowcat to enable it")
```

Comment:
These seems like noise to me.

The option is documented in the help. I don't think we want to tell the user about every option they're *not* using in the logs.

Reply:
Yeah. This message is only for the short transition period.